src/courseware/course/Course.test.jsx

```diff
@@ -280,8 +280,8 @@ describe('Course', () => {
 
     loadUnit();
     await waitFor(() => expect(screen.queryByText('Loading learning sequence...')).not.toBeInTheDocument());
-    screen.getAllByRole('button', { name: /previous/i }).forEach(button => fireEvent.click(button));
-    screen.getAllByRole('button', { name: /next/i }).forEach(button => fireEvent.click(button));
+    screen.getAllByRole('link', { name: /previous/i }).forEach(button => fireEvent.click(button));
+    screen.getAllByRole('link', { name: /next/i }).forEach(button => fireEvent.click(button));
```

Comment:
functionally identical but just for naming consistency

```suggestion
    screen.getAllByRole('link', { name: /previous/i }).forEach(link => fireEvent.click(link));
    screen.getAllByRole('link', { name: /next/i }).forEach(link => fireEvent.click(link));
```